examples/funloc/funloc_params.yml

```diff
@@ -52,9 +53,12 @@ preprocessing:
     phase: zero
   ssp:
     proj_nums: {
-      subj_01: [[2, 2, 0], [1, 1, 3], [0, 0, 0], [1, 1, 0], [1, 1, 0]],
-      subj_02: [[2, 2, 0], [1, 1, 2], [0, 0, 0]],
+      subj_01: [[2, 2, 0], [1, 1, 3], [8, 8, 0], [1, 1, 0], [1, 1, 0]],
+      subj_02: [[2, 2, 0], [1, 1, 2], [8, 8, 0]],
       }
+    erm_proj_lp_cut: 20
+    erm_proj_lp_cut: 40
```

Comment:
Probably meant `erm_proj_hp_cut: 20` and lp cut at 40.

Reply:
Yikes this made me realize that the code was wrong, and that the band-passing was almost certainly not doing what it was supposed to. I also reused `p.cont_lp` and `p.cont_hp` because they already existed. Feel free to look at the updated docs/params and give it a shot

Reply:
I plan to make a run through with a good subject example soon, and I'll let you know if I come across a bug or have questions about the algorithm (especially how the revised SSS step relates to subsequent SSP processing).